_posts/2024-12-11-release-0.10.0.md

```diff
@@ -0,0 +1,150 @@
+
+## Stunning new environments and assets
+
+### Revamped Town 10
+
+The enhanced geometry handling system underpinning CARLA opens the door to rendering environments with much higher polygon counts and more realistic lighting. To fully exploit this, this version of CARLA presents a revamped Town 10 with brand new buildings, pavements, roads and vegetation all organized around the same familiar road network. 
+
+![town10_montage](/img/posts/2024-12-11/town10_montage.webp){:class="img-fluid rounded mx-auto d-block"}
+
+The newly remodeled town includes impressive new towerblocks with dazzling reflective glass, including meticulously detailed retail units on the lowest floors. The extra detail permitted by the new geometry system really shows in the magnificent classically styled buildings found in several places in the map.
+
+![town10_classical](/img/posts/2024-12-11/town10_classical_buildings.webp){:class="img-fluid rounded mx-auto d-block"}
+
+Additionally, Town 10 introduces some new construction assets, giving the impression of an evolving cityscape.
+
+![town10_construction](/img/posts/2024-12-11/town10_construction.webp){:class="img-fluid rounded mx-auto d-block"}
+
+**Enhanced road topology**
+
+The enhancements in detail in Town 10 extend right down to the road surface itself. Town 10's roads now have a realistic camber and sporadic road surface variations such as potholes and speed bumps which impact not only the appearance, but also vehicle dynamics and sensors, further enhancing simulation realism. 
+
+![town10_road_surface](/img/posts/2024-12-11/town10_potholes_speedbumps.webp){:class="img-fluid rounded mx-auto d-block"}
+
+### Upgraded vehicle roster
+
+On top of Town 10's makeover, version 0.10.0 comes with a roster of 11 beautifully remodeled vehicles, exploiting the higher geometry budget available and improved lighting to capture every detail with enhanced definition. 
+
+![vehicle_montage](/img/posts/2024-12-11/vehicle_montage.webp){:class="img-fluid rounded mx-auto d-block"}
+
+Unreal Engine 5 also introduces the new [Chaos physics engine](https://dev.epicgames.com/documentation/en-us/unreal-engine/chaos-physics-overview?application_version=4.27) with additional options for vehicle dynamics modeling. All vehicles included in CARLA 0.10.0 now use the new engine, adding more realistic modeling of suspension behavior.
+
+![vehicle_suspension](/img/posts/2024-12-11/vehicle_physics_speed_bumps.webp){:class="img-fluid rounded mx-auto d-block"}
+
+### New off-road mine map from Synkrotron
+
+With release 0.10.0, CARLA goes off-road with a spectacular open-cast mine map, provided by CARLA consortium member and sponsor [**Synkrotron**](https://www.synkrotron.ai/). The map includes a parking area connected to the mine area through a small network of access roads. 
```

Comment:
Can we add Synkrotron logo here, in the same way as InvertedAI's logo in the following paragraph?
![logo1](https://github.com/user-attachments/assets/ccbee61f-b672-4aa0-9e70-6a8ef854ccd6)

Reply:
Sure, that's a good idea:
![image](https://github.com/user-attachments/assets/03c429fc-1668-4900-9403-13e8d52d0bcb)

Reply:
Thanks, Matt!